app/src/main/java/com/es/findsoccerplayers/ActivitySettings.java

```diff
@@ -16,5 +27,32 @@ protected void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
+        settElemList.add(new SettingsElement(R.drawable.ic_account,"Account","View and modify account options"));
+        settElemList.add(new SettingsElement(R.drawable.ic_edit_location_24,"Edit search range","Modify the searching distance for the available matches"));
+        settElemList.add(new SettingsElement(R.drawable.ic_log_out_24,"Log out","Disconnect from your account"));
+
+        RecyclerView mRecyclerView = findViewById(R.id.settings_recyclerview);
+        mRecyclerView.setHasFixedSize(true);
+        LinearLayoutManager mLayoutManager = new LinearLayoutManager(this);
+        SettingsAdapter mAdapter = new SettingsAdapter(settElemList);
+
+        mRecyclerView.setLayoutManager(mLayoutManager);
+        mRecyclerView.addItemDecoration(new DividerItemDecoration(mRecyclerView.getContext(), DividerItemDecoration.VERTICAL));
+        mRecyclerView.setAdapter(mAdapter);
+
+        mAdapter.setOnItemClickListener(new SettingsAdapter.OnItemClickListener() {
+            @Override
+            public void onItemClick(int position) {
+                settElemList.get(position);
+                if(position == 0){
+                    startActivity(new Intent(getApplicationContext(), ActivityAccount.class));
+                }else if(position == 1){
+                    startActivity(new Intent(getApplicationContext(), ActivitySetLocation.class));
+                }else if(position == 2){
+                    Utils.showUnimplementedToast(getApplicationContext());
```

Comment:
in realtà il log out funziona ed è implementato

Reply:
togliamo il log out nell'altra activity di modifica del profilo ed eseguiamo il codice del log out quando uno clicca su "log out" appunto nella lista delle impostazioni

Reply:
si ma non sapendo se lasciavamo il log out o implementavamo qualcos'altro ho messo TODO

Reply:
vabbeh dai lasciamo cosi per il momento